kong/cmd/utils/migrations.lua

```diff
@@ -128,6 +128,15 @@ local function up(schema_state, db, opts)
       error("database has pending migrations; run 'kong migrations finish'")
     end
 
+    if opts.force then
+      log.debug("forcing re-execution of these migrations:\n%s",
+                schema_state.executed_migrations)
+
+      assert(db:run_migrations(schema_state.executed_migrations, {
+        run_up = true,
+      }))
```

Comment:
The error messages following this branch would look slightly weird imho. E.g.:
```
forcing re-execution of these migrations: [...]
...
...
database is already up-to-date
```

And:
```
forcing re-execution of these migrations: [...]
...
...
migrations to run: [...]
```

It would be enough to slightly update these logs so that their text makes sense in both contexts (forced and not forced migrations). Or to wrap them in appropriate branches determining if the `--force` flag is in use.

Reply:
That module already had 2 `can't`s and 2 `cannot`s in other messages :) Will standardize them on `cannot` for :robot: computer voice :robot:  reasons, then

Reply:
regarding the messages, I think just removing the `database is already up-to-date` message in the forced case would be enough.

Reply:
Yep, sounds good to me!

Reply:
updated